pkg/cmd/cli/install/install.go

```diff
@@ -143,6 +143,8 @@ This is useful as a starting point for more customized installations.
 
 	# velero install --bucket gcp-backups --provider gcp --secret-file ./gcp-creds.json --wait
 
+	# velero install --bucket backups --provider aws --backup-location-config region=us-west-2 --snapshot-location-config region=us-west-2 --pod-annotations iam.amazonaws.com/role=arn:aws:iam::<AWS_ACCOUNT_ID>:role/<VELERO_ROLE_NAME>
```

Comment:
@tlkamp Question for your usage - do you specify a `--secret-file` argument? I notice that #1626 we missed that it's a required argument currently. We might have to do another change where we require either the secret file or annotations.

Reply:
@nrb we do since it's required, but we are simply sending an empty file (it isn't a big deal for automation since there are no contents to keep in sync 😉 ). I wasn't sure how best to handle this situation in my PR since the presence of additional annotations doesn't necessarily mean another auth. mechanism is being used. 

I think making the `--secret-file` optional and detailing _when_ to provide it (and what to provide) is probably the best happy-medium that can be reached.

Reply:
Awesome, thanks for that input. I agree with that assessment, so I’ll see about making that change.

Reply:
Another idea I had while thinking about this is to introduce a 2nd flag that explicitly says not to create a secret. Something like `--no-secret` or `--other-auth`.

This way, we don't need to create an empty secret, and it's saying the auth will be handled in some other way without making an assumption about what's in annotations.

I would think this would mean we'd require one of `--secret-file` or this new argument, not both. I also don't think it's relevant to the Helm chart, as it has ways of controlling whether to create secrets or not already.

Reply:
@nrb that's indeed an option! Thinking back to what @skriss said about wanting to keep flags limited, I think just making `--secret-file` optional achieves the same functionality in a single flag.

Reply:
xref #1689